List of Commcare application ports information

prem-fissionhq opened the pull request:
<!--- Provide a link to the ticket or document which prompted this change -->

##### ENVIRONMENTS AFFECTED
<!--- list which environments are affected by this change or None if this doesn't change any environment files -->

esoergel commented:
Hey Prem, could you move that table into the ["Prerequisites" section of the new environment doc](https://github.com/dimagi/commcare-cloud/blob/master/docs/setup/new_environment.md#prerequisites)?  Here's an example of a table in markdown:
https://github.com/dimagi/commcare-cloud/blob/master/docs/howto/transfer-domain.md#1-switch-mobile-devices-to-a-proxy-url

Then could you add those IDs and descriptions to the script and instruct the developer to run that script in that same section?

prem-fissionhq commented:
Hello Ethan,
As per your suggestions, I have Converted CSV to markdown table and included the link in the new_environment.md file. Also, updated the script to provide a port description for the ports which are not accessible. 
Please review and let me know if any changes are required. Thanks
